openapi/openapi.yaml

```diff
@@ -405,15 +405,1608 @@ components:
             - constraints
           properties:
             requirements:
-              type: object
+              type: string
               description: >-
                 Pipfile stating software stack (direct dependencies) of an
                 application.
-              additionalProperties: {}
+              example: >-
+                [[source]]
+                url = "https://pypi.org/simple"
+                verify_ssl = true
+                name = "pypi"
+
+                [packages]
```

Comment:
It might be a good idea to keep this simple - some stack with one simple package that prints some "Hello world" might work.